content/applications/marketing/marketing_automation/advanced/understanding_metrics.rst

```diff
@@ -52,7 +56,7 @@ which always keeps the flow up-to-date.
 
 .. tip::
    Filters can also be applied to individual activities, in the :guilabel:`Domain` section of the
-   activity pop-up form. Use this feature to specify an individual filter that will only be
+   activity pop-up form. Use this feature to specify an individual filter that will *only* be
    performed if the records satisfy *both* filter criteria - the activity filter and its specific
    domain filter.
 
```

Comment:
I think this should be an example admonition
```suggestion
.. example::
   When configuring an SMS activity, make sure a certain phone number is set, in order to avoid
   triggering a SMS that would never be sent.
```